Gemfile

```diff
@@ -12,32 +12,20 @@ gem "handsoap", "~>0.2.5", :require => false, :git => "https://github.com/Manage
 gem "rubywbem",            :require => false, :git => "https://github.com/ManageIQ/rubywbem.git", :branch => "rubywbem_0_1_0"
 
 # when using this Gemfile inside a providers Gemfile, the dependency for the provider is already declared
-unless dependencies.detect { |d| d.name == "manageiq-providers-openstack" }
-  gem "manageiq-providers-openstack", :git => "https://github.com/ManageIQ/manageiq-providers-openstack", :branch => "master"
-end
-unless dependencies.detect { |d| d.name == "manageiq-providers-hawkular" }
-  gem "manageiq-providers-hawkular", :git => "https://github.com/ManageIQ/manageiq-providers-hawkular", :branch => "master"
-end
-unless dependencies.detect { |d| d.name == "manageiq-content" }
-  gem "manageiq-content", :git => "https://github.com/ManageIQ/manageiq-content", :branch => "master"
-end
-unless dependencies.detect { |d| d.name == "manageiq-providers-amazon" }
-  gem "manageiq-providers-amazon", :git => "https://github.com/ManageIQ/manageiq-providers-amazon", :branch => "master"
-end
-unless dependencies.detect { |d| d.name == "manageiq-providers-azure" }
-  gem "manageiq-providers-azure", :git => "https://github.com/ManageIQ/manageiq-providers-azure", :branch => "master"
-end
-unless dependencies.detect { |d| d.name == "manageiq-providers-vmware" }
-  gem "manageiq-providers-vmware", :git => "https://github.com/ManageIQ/manageiq-providers-vmware", :branch => "master"
-end
-
-unless dependencies.detect { |d| d.name == "manageiq-ui-classic" }
-  gem "manageiq-ui-classic", :git => "https://github.com/ManageIQ/manageiq-ui-classic", :branch => "master"
+def miq_plugin(plugin_name)
+  unless dependencies.detect { |d| d.name == plugin_name }
+    gem plugin_name, :git => "https://github.com/ManageIQ/#{plugin_name}", :branch => "master"
+  end
 end
 
-unless dependencies.detect { |d| d.name == "manageiq-providers-lenovo" }
-  gem "manageiq-providers-lenovo", :git => "https://github.com/ManageIQ/manageiq-providers-lenovo", :branch => "master"
-end
+miq_plugin "manageiq-content"
+miq_plugin "manageiq-providers-amazon"
+miq_plugin "manageiq-providers-azure"
+miq_plugin "manageiq-providers-hawkular"
+miq_plugin "manageiq-providers-lenovo"
+miq_plugin "manageiq-providers-openstack"
+miq_plugin "manageiq-providers-vmware"
+miq_plugin "manageiq-ui-classic"
```

Comment:
We have some more repo splits coming, so let's wait on this until after those are in.

Also, I'd prefer `manageiq_plugin` over miq